colossalai/pipeline/rpc/_pipeline_base.py

```diff
@@ -230,6 +220,26 @@ def get_output_by_key(self, key: UniqueKey, ref_use=False) -> Any:
 
         return output
 
+    def sync_global_worker_rrefs(self, pp_rank_to_worker_rref: Dict[int, PyRRef]) -> None:
```

Comment:
Why this function moves to here?

Reply:
Because I want to put all private functions which start with '_' together, seperated from public functions.

Reply:
I see